app/views/podcasts/show.rss.builder

```diff
@@ -59,7 +59,7 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
     end
 
     xml.itunes :subtitle, @podcast.subtitle unless @podcast.subtitle.blank?
-    xml.itunes(:summary) { xml.cdata!(@podcast.summary) } unless @podcast.summary.blank?
+    xml.itunes(:summary) { xml.cdata!(itunes_summary(@podcast)) }
```

Comment:
good question. hmm. I was thinking that between desc and summary we would always have something to put in here, since for a podcast description is a required attribute, but you know, people don't follow the rules, so I'm going to put a check back in.